src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -114,7 +114,7 @@ protected override Task UpdateDisplayAsync(in RenderBatch batch)
             if (_disposing)
             {
                 // We are being disposed, so do no work.
-                return Task.FromCanceled<object>(CancellationToken.None);
+                return CanceledTask;
```

Comment:
Was there a repro for this issue ? or just an random improvment?

Reply:
I was messing around with circuit expiry and saw the ArgOutOfRange exception pop up in the console. I haven't super investigated it

Reply:
Sounds good, and this is the exactly the kind of issues we should fixing during the reliablity/capacity/perf push.